pkg/deployments/tasks/20220721-4626-factory/index.js

```diff
@@ -0,0 +1,117 @@
+const { task } = require("hardhat/config");
+const data = require("./input");
+
+const { SENSE_MULTISIG } = require("../../hardhat.addresses");
+
+const dividerAbi = require("./abi/Divider.json");
+const peripheryAbi = require("./abi/Periphery.json");
+const { delay, verifyOnEtherscan } = require("../../hardhat.utils");
+
+task(
+  "20220721-4626-factory",
+  "Deploys 4626 Factory and adds it to the Periphery. Also deploys Sense Master & Chainlink Oracles",
+).setAction(async ({}, { ethers }) => {
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+  const chainId = await getChainId();
+  const deployerSigner = await ethers.getSigner(deployer);
+
+  console.log(`Deploying from ${deployer} on chain ${chainId}`);
+
+  const { divider: dividerAddress, periphery: peripheryAddress, factories, maxSecondsBeforePriceIsStale } = data[hre.network.name] || data.mainnet;
+  let divider = new ethers.Contract(dividerAddress, dividerAbi, deployerSigner);
+  let periphery = new ethers.Contract(peripheryAddress, peripheryAbi, deployerSigner);
+
+  console.log("\n-------------------------------------------------------");
+  console.log("\nDeploy Sense Chainlink Price Oracle");
+  const { address: chainlinkOracleAddress } = await deploy("ChainlinkPriceOracle", {
+    from: deployer,
+    args: [maxSecondsBeforePriceIsStale],
+    log: true,
+  });
+
+  console.log("\n-------------------------------------------------------");
+  console.log("\nDeploy Sense Master Price Oracle");
+  const { address: masterOracleAddress } = await deploy("MasterPriceOracle", {
+    from: deployer,
+    args: [chainlinkOracleAddress, [], []],
+    log: true,
+  });
+
+  console.log("\n-------------------------------------------------------");
+  console.log("\nDeploy Factories");
+  for (let factory of factories) {
+    const {
+      contractName: factoryContractName,
+      ifee,
+      stake,
+      stakeSize,
+      minm,
+      maxm,
+      mode,
+      tilt,
+    } = factory;
+
+    oracle = masterOracleAddress;
+    console.log(
+      `\nDeploy ${factoryContractName} with params ${JSON.stringify({
+        ifee: ifee.toString(),
+        stake,
+        stakeSize: stakeSize.toString(),
+        minm,
+        maxm,
+        mode,
+        oracle,
+        tilt
+      })}}`,
+    );
+    const factoryParams = [oracle, stake, stakeSize, minm, maxm, ifee, mode, tilt];
```

Comment:
very nit:
```suggestion
    const factoryParams = [masterOracleAddress, stake, stakeSize, minm, maxm, ifee, mode, tilt];
```
& remove extra `oracle = masterOracleAddress;`?